_dev/.eslintrc

```diff
@@ -1,12 +1,17 @@
 {
   "extends": ["plugin:fxa/recommended"],
   "plugins": ["fxa"],
+  "parser": "@typescript-eslint/parser",
   "parserOptions": {
     "ecmaVersion": "2020"
   },
   "rules": {
     "require-atomic-updates": "off",
     "space-unary-ops": "off",
-    "no-useless-escape": "off"
+    "no-useless-escape": "off",
+    "no-unused-vars": "off",
+    "@typescript-eslint/no-unused-vars": [2, {"vars": "all", "args": "none"}],
+    "no-redeclare": "off",
+    "@typescript-eslint/no-redeclare": "error"
```

Comment:
Outdated file

Reply:
Okay thanks. This question still applies to `packages/fxa-auth-server/.eslintrc.json` rule changes though.

Reply:
Also, AFAICT, we're not using `_dev/.eslintrc` anywhere -- so can we remove that file entirely now? It's not going to get used automatically based on https://eslint.org/docs/latest/user-guide/configuring/configuration-files#cascading-and-hierarchy, since `_dev` is not a parent directory of any of the files being linted.

Reply:
Will happen in 6137, yep - already have that implemented in my local branch. I don't necessarily want to move this to a JS file.

Reply:
Added a comment to https://mozilla-hub.atlassian.net/browse/FXA-6137?focusedCommentId=609569 to take care of this.